client/state/sites/reducer.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { omit, merge, get, includes, reduce, isEqual, stubFalse, stubTrue } from 'lodash';
```

Comment:
Glad to see those lodash fns removed 👏  Many of those in particular are needlessly used throughout our reducer code.

Reply:
I knew you're going to like this! 🙂